wallet/src/wallet/mod.rs

```diff
@@ -97,6 +108,7 @@ pub struct Wallet<B: storage::Backend> {
     db: Store<B>,
     key_chain: MasterKeyChain,
     accounts: BTreeMap<U31, Account>,
+    unsynced_accounts: BTreeMap<U31, Account>,
```

Comment:
Apologies for the confusion. But this was an Option and the reasoning was that it can only be one "unsynced account". Has that changed now?

Reply:
It was to fix the case when the wallet process is killed in the middle of syncing the accounts with new blocks, if some accounts are synced and the rest not, loading up the wallet can have multiple unsynced accounts.